lib/apipie/extractor/recorder.rb

```diff
@@ -160,6 +160,7 @@ def process(*, **) # action, parameters = nil, session = nil, flash = nil, http_
         ensure
           Apipie::Extractor.clean_call_recorder
         end
+        ruby2_keywords :process if respond_to?(:ruby2_keywords, true)
```

Comment:
Can you explain to us why you added this? And why it causes a warning?

Thanks

Reply:
See https://github.com/Apipie/apipie-rails/pull/716/commits/684cd533d27261b36b7cfea06584191f39eabafa

I think what happened is https://github.com/Apipie/apipie-rails/commit/a55d836f4e8609395c374d85b2542a3ac7277a2c got merged, but the `process` params should be

```
        def process(*) # action, parameters = nil, session = nil, flash = nil, http_method = 'GET')
          ret = super
```
per https://eregon.me/blog/2021/02/13/correct-delegation-in-ruby-2-27-3.html

Reply:
Thanks @hank-spokeo! I'll create a PR to fix this.

Reply:
Thanks a lot @hank-spokeo